packages/datastore/src/datastore.ts

```diff
@@ -111,9 +108,12 @@ class Datastore implements IIterable<Table<Schema>>, IMessageHandler, IDisposabl
     this._disposed = true;
     Signal.clearData(this);
 
-    this._broadcastHandler = null;
+    this._adapter = null;
```

Comment:
Should the adapter be disposed here, now that the datastore-adapter relation is one-way only?

Reply:
Hmm, I typically think that the entity that creates an object should be responsible for disposing of it.

Also, still willing to be convinced that it shouldn't be one-way only :)